bridge/node/README.md

```diff
@@ -0,0 +1,90 @@
+# DSNP Graph SDK
```

Comment:
Nice doc. Might just want to call out that interacting with the chain (fetching graph to import, committing exported graph changes to the chain) would be done using another package such as `@polkadot-js`

Reply:
Yes intend to do it with #124 PR which will have full example/tests etc.